stablehlo/dialect/VhloBytecode.cpp

```diff
@@ -216,6 +216,14 @@ enum TypeCode {
   ///   }
   kFloatF64V1Type = 5,
 
+  ///   FloatF8E3M4V1Type {
+  ///   }
+  kFloatF8E3M4V1Type = 36,
+
+  ///   FloatF8E4M3V1Type {
+  ///   }
+  kFloatF8E4M3V1Type = 35,
```

Comment:
At the top of this list there is a "next available code is 35" comment, can you update that too

Reply:
fixed